packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -154,7 +154,7 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
     const { rootProps, ariaDescribedBy, ariaLabelledBy, className } = this.props;
     const divProps = getNativeProps(this.props, htmlElementProperties);
 
-    const Tag = this.props.elementType || 'div';
+    const Tag = this.props.as || 'div';
```

Comment:
We need to deprecate this but still fall back to it until we remove it entirely. This should be something like:
```tsx
const Tag = this.props.as || this.props.elementType || 'div';
```

Also need to add a call to `_warnDeprecations` which you can find used throughout the codebase. If this breaks any tests you'll have to move them to a `deprecated.test` file which disables `_warnDeprecations` so tests don't fail (of which there are also examples of in the codebase.)

Reply:
Also need to mark `as` prop with `@deprecated Replaced with elementType`.